src/data/mainnet/ethereum-tokens-info.json

```diff
@@ -36,7 +37,9 @@
     "address": "0xdac17f958d2ee523a2206206994597c13d831ec7",
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/USDT.png",
     "infoUrl": "https://www.coingecko.com/en/coins/tether",
-    "minimumAppVersionToSwap": "1.72.0"
+    "minimumAppVersionToSwap": "1.72.0",
+    "isStableCoin": true,
```

Comment:
Are there any side effects of marking these with `isStableCoin`? IIRC we don't do this for anything but the c-stables at the moment. Wondering if there are assumptions in the app that `isStableCoin: true` implies that the coin is a c-stable.

Reply:
We already have it for USDC actually

Reply:
The only current use I can find in the wallet is in [TokenDetails](https://github.com/valora-inc/wallet/blob/main/src/tokens/TokenDetails.tsx#L99)

We hide the +/- price differentials for stables

Reply:
and other older versions of the wallet won't get the ethereum tokens anyways

Reply:
ohhh okay, cool -- somehow missed we already had it on USDC.